vnext/ReactWindowsCore/MemoryMappedBuffer.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include "jsi/jsi.h"
+
+#include <memory>
+
+namespace Microsoft::JSI {
+
+// We only support files whose size can fit within an uint32_t. Memory
+// mapping an empty or a larger file fails.
+std::shared_ptr<facebook::jsi::Buffer> makeMemoryMappedBuffer(
```

Comment:
Pascal convention, please.
`MakeMemoryMappedBuffer`.

Reply:
Fixed.

---
In reply to: [320977423](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320977423) [](ancestors = 320977423)